parseable/benchmark.sh

```diff
@@ -0,0 +1,38 @@
+# Install Dependencies
+sudo apt-get update -y
+sudo apt-get install -y rustc rustup
+rustup update stable
+
+
+# Clone Parseable Repository and checkout v1.7.3 branch
+git clone https://github.com/parseablehq/parseable.git
+cd parseable
+git checkout v1.7.3
+
+# Build and Run Parseable
+cargo build --release
+export PATH="`pwd`/target/release:$PATH"
+export RUST_LOG=debug
+./target/release/parseable local-store > parseable.log 2>&1 & PARSEABLE_PID=$!
+# Verify Parseable is running
+if ps -p $PARSEABLE_PID > /dev/null; then
+    echo "Parseable is running with PID: $PARSEABLE_PID"
+else
+    echo "Error: Parseable failed to start. Check parseable.log for details."
+    exit 1
+fi
+
+#run ingestion script
+./ingestion.sh
```

Comment:
Scripts `./ingestion.sh` and `./run_query.sh` will need the executable `x` bit to be able to run.